openapi/src/parts/workspace.yaml

```diff
@@ -418,11 +418,7 @@ components:
           description: ID of provided spend profile
           type: string
         policies:
-          description: Policies provided by the containing folder; NOT IMPLEMENTED
-          type: array
-          items:
-            type: string
-            format: uuid
+          $ref: '#/components/schemas/TpsPolicyInputs'
```

Comment:
Maybe we should just get rid of this until we implement it. The input might want to be the UUID of the "folder" (or whatever we call the container of workspaces.

Reply:
Right now this isn't used by callers and I think that will probably be true while we figure out the exact shape of policy. I'm inclined to leave this in as a best-guess for now and be really cautious about relying on it heavily in clients until folders/LZs are a bit clearer, but it seems reasonable to me that we would have policy on a workspace beyond its folder restrictions